project/src/main/java/com/google/sps/servlets/SignUpServlet.java

```diff
@@ -3,6 +3,8 @@
 import com.google.appengine.api.images.ServingUrlOptions;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
```

Comment:
Unused imports?

Reply:
Deleted them! I was going to create a new property and then changed my mind but forgot to remove the imports..